### ISSUE TITLE
Use resque-scheduler instead of whenever/cron

### ISSUE THREAD

**Issue by mconf-daileon** [mconf/bigbluebutton_rails#102]
---

Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Redmine Issue: 1167, http://dev.mconf.org/redmine/issues/1167

Original Assignee: Leonardo Daronco

---

Whenever is a gem used to ease the configuration of cron tasks. BigbluebuttonRails currently uses cron tasks to run two processes:
* Update the list of recordings;
* Find out when meetings ended.

We are already using resque to monitor when meetings start, so we could replace whenever/cron by "resque-scheduler":https://github.com/resque/resque-scheduler, a gem that makes resque work similarly as cron.

This would reduce the number of times the application has to be bootstrapped, since resque-scheduler loads the environment only once and not every time you run a task (as happens when using cron).

It would also reduce the number of configurations and dependencies needed to use the gem. Whenever would not be a dependency anymore, and the configuration of resque that is already done wouldn't change.

See more at: http://www.perfectline.co/2011/07/cron-tasks-for-your-rails-application-with-resque-2/

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Date: 2015-03-26T17:21:52Z

---

Already in use in the current Mconf-Web (in production as well) and working fine.
